frontend/src/app/application-forms/tree-application-form/tree-permit-view/tree-permit-view.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, SecurityContext } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Title } from '@angular/platform-browser';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Component({
+  selector: 'app-tree-permit-view',
+  templateUrl: './tree-permit-view.component.html'
+})
+export class TreePermitViewComponent implements OnInit {
+  forest: any;
+  permit: any = { totalCost: 0, quantity: 0, emailAddress: '' };
```

Comment:
this feels like a stylistically a strange way to initialize this view. is this what is recommended by angular?

Reply:
We didn't need to define default values there. Removed initialization on the permit.